glib/src/gobject/binding.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <https://opensource.org/licenses/MIT>
+use crate::translate::*;
```

Comment:
Please add an empty line after the license comment. :)

Reply:
Done